milestones/services.py

```diff
@@ -0,0 +1,44 @@
+"""
+A wrapper class around all methods exposed in api.py
```

Comment:
This docstring confuses me.
- There are definitely more methods in `api.py` than there are in `REQUESTED_FUNCTIONS`, so that needs to be rectified.
- We claim to link to `api.py`, but that filename doesn't appear anywhere in this module, and it seems like we're counting on the caller to provide the correct API. Is that a standard pattern @nasthagiri?